jena-arq/testing/RIOT/URISchemes/app.nt

```diff
@@ -0,0 +1 @@
+<app://2dee5b0a-6100-470a-a67f-1399518cb470/nested/foo.txt> <http://www.w3.org/2000/01/rdf-schema#seeAlso> <app://2dee5b0a-6100-470a-a67f-1399518cb470/bar.txt> .
```

Comment:
N-Triples is handled differently - there is no IRI base resolution in normal parsing; this is intentional.

The command line "riot" specially adds IRI checking for N-Triples. Please remove the NT tests.

Reply:
I included this to verify that the IRIs were acceptable for parsing overall, precisely as N-Triples has no base resolution.

Reply:
Should I still remove `*.nt`? For all we know future code might use different iri factories for relative resolution and NT/NQ parsing, as all the other formats use `file:///current/directory` as base by default.

Reply:
Removed *.nt tests in 5200369653704be86e277774e6ce6d856369eec6